libs/application/templates/estate/src/forms/Sections/description.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  buildDescriptionField,
+  buildSection,
+} from '@island.is/application/core'
+
+import { m } from '../../lib/messages'
+import { EstateTypes } from '../../lib/constants'
+
+export const applicationDescription = buildSection({
+  id: 'externalData',
+  title: m.applicationDescriptionSectionTitle,
+  children: [
+    buildDescriptionField({
+      id: 'applicationDescription',
+      space: 2,
+      title: (application) =>
+        application.answers.selectedEstate === EstateTypes.estateWithoutAssets
+          ? /* EIGNALAUST DÁNARBU */
+            m.applicationDescriptionTitleEstateWithoutAssets
+          : application.answers.selectedEstate === EstateTypes.officialDivision
+          ? /* OPINBER SKIPTI */
+            m.applicationDescriptionTitleOfficialDivision
+          : application.answers.selectedEstate ===
+            EstateTypes.permitForUndividedEstate
+          ? /* SETA Í ÓSKIPTU BÚI */
+            m.applicationDescriptionTitleUndividedEstate
+          : /* EINKASKIPTI */
+            m.applicationDescriptionTitleDivisionOfEstateByHeirs,
+      description: (application) =>
+        application.answers.selectedEstate === EstateTypes.estateWithoutAssets
+          ? /* EIGNALAUST DÁNARBU */
+            m.applicationDescriptionTextEstateWithoutAssets
+          : application.answers.selectedEstate === EstateTypes.officialDivision
+          ? /* OPINBER SKIPTI */
+            m.applicationDescriptionTextOfficialDivision
+          : application.answers.selectedEstate ===
+            EstateTypes.permitForUndividedEstate
+          ? /* SETA Í ÓSKIPTU BÚI */
+            m.applicationDescriptionTextUndividedEstate
+          : /* EINKASKIPTI */
+            m.applicationDescriptionTextDivisionOfEstateByHeirs,
+    }),
```

Comment:
Would want to see this text more ready if we're opening on these applications next Tuesday, since two of these are already live.

Reply:
text added to contentful
![image](https://github.com/island-is/island.is/assets/78020509/c546fd4c-4ffe-4a0e-bcc8-27d503ff797a)